contracts/governance/Governor.sol

```diff
@@ -327,6 +327,20 @@ abstract contract Governor is Context, ERC165, EIP712, IGovernor, IERC721Receive
         return proposalId;
     }
 
+    /**
+     * @dev See {IGovernorTimelock-queue}.
```

Comment:
I'm a bit confused. I think this NatSpec doesn't clear explain why this function is right here. Even if it's for Timelock modules, why are adding this function instead of the `IGovernorTimelock`? or, why can't we just declare it as a virtual function if the contract is already abstract?

Reply:
This was moved to `IGovernor`